composer/airflow_1_samples/constraints.txt

```diff
@@ -1,3 +1,4 @@
 pandas-gbq==0.14.1 # must be under 0.15.0 until https://github.com/apache/airflow/issues/15113 is addressed
 SQLAlchemy==1.3.23 # must be under 1.4 until at least Airflow 2.0 (check airflow setup.py for restrictions)
-WTForms==2.3.3 # Must stay under 3.0 because of https://github.com/dpgaspar/Flask-AppBuilder/issues/1732
\ No newline at end of file
+WTForms==2.3.3 # Must stay under 3.0 because of https://github.com/dpgaspar/Flask-AppBuilder/issues/1732
+MarkupSafe==1.1.1 # as found in Airflow 1.10.15 constraints
```

Comment:
This is conflicting with the `markupsafe==2.0.1` pinned in the `composer/airflow_1_samples/requirements.txt` file, could you revert it on that file?

Reply:
Ooops, great catch. I'll fix that!

Reply:
fixed!